packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx

```diff
@@ -48,14 +52,42 @@ export default function OrganizationGenerationInformationTile({
   datacenterCount = 0,
 }: TTileProps) {
   const { t } = useTranslation('dashboard');
+  const [showModal, setShowModal] = useState<TShowModal>(false);
 
   return (
     <OsdsTile className="w-full h-full flex-col" inline rounded>
+      {showModal === 'name' && (
+        <EditNameModal
+          organizationName={vcdOrganization?.currentState?.fullName}
+          onCloseModal={() => setShowModal(false)}
+          onEdit={() => {}}
+        />
+      )}
+      {showModal === 'description' && (
+        <EditDescriptionModal
+          organizationDescription={vcdOrganization?.currentState?.fullName}
+          onCloseModal={() => setShowModal(false)}
+          onEdit={() => {}}
+        />
+      )}
```

Comment:
Modals have their own route and urls :
https://github.com/ovh/manager/blob/master/packages/manager/apps/pci-rancher/src/routes.tsx

Take example on the pci-rancher project with the EditModal or GenerateAccessModal